goodcheck.gemspec

```
@@ -20,14 +20,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 2.4.0'
+  spec.required_ruby_version = '>= 2.5.0'
 
   spec.add_development_dependency "bundler", ">= 1.16"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest", "~> 5.0"
   spec.add_development_dependency "minitest-reporters", "~> 1.3.6"
 
-  spec.add_runtime_dependency "activesupport", ">= 4.0", "< 6.0"
+  spec.add_runtime_dependency "activesupport", ">= 4.0", "~> 6.0.0"
```

I prefer `>= x, < y` style over `~>` because the latter style is confusing. What do you think about this?

```suggestion
  spec.add_runtime_dependency "activesupport", ">= 4.0", "< 7.0"
```

---

Either is fine!! Should we wait `soutaro` 's answer??

---

Yeah, let's wait for a while.